app/src/main/java/com/android/sample/resources/C.kt

```diff
@@ -8,6 +8,11 @@ object C {
 
     const val main_screen_container = "main_screen_container"
 
+    // General
+    const val LOADING = "Loading..."
+    const val PADDING = 16
+    const val SMALL_PADDING = 8
```

Comment:
Very good point, now we can have consistent padding! :)

Reply:
Thanks you !